--- ghostpdl-9.19/xps/xps.mak.orig	2016-03-23 09:19:58.000000000 +0100
+++ ghostpdl-9.19/xps/xps.mak	2016-08-01 21:49:32.874974496 +0200
@@ -57,7 +57,7 @@
 $(XPSOBJ)xpsjpeg.$(OBJ): $(XPSSRC)xpsjpeg.c $(XPSINCLUDES) $(XPS_MAK) $(MAKEDIRS)
 	$(XPSCCC) $(XPSSRC)xpsjpeg.c $(XPSO_)xpsjpeg.$(OBJ)
 
-$(XPSOBJ)xpspng.$(OBJ): $(XPSSRC)xpspng.c $(XPSINCLUDES) $(png__h) \
+$(XPSOBJ)xpspng.$(OBJ): $(XPSSRC)xpspng.c $(XPSINCLUDES) \
                         $(PNGGENDIR)$(D)libpng.dev $(XPS_MAK) $(MAKEDIRS)
 	$(XPSCCC) $(I_)$(PNGSRCDIR)$(_I) $(XPSSRC)xpspng.c $(XPSO_)xpspng.$(OBJ)
 
