--- ghostpdl-9.54.0/base/ugcclib.mak.orig	2021-04-26 19:52:50.230050484 +0200
+++ ghostpdl-9.54.0/base/ugcclib.mak	2021-04-26 19:55:35.098602228 +0200
@@ -96,10 +96,7 @@ GCFLAGS_NO_WARN=-fno-builtin -fno-common
 GCFLAGS_WARNINGS=-Wall -Wcast-qual -Wpointer-arith -Wstrict-prototypes -Wwrite-strings
 GCFLAGS=$(GCFLAGS_NO_WARN) $(GCFLAGS_WARNINGS)
 XCFLAGS=
-CFLAGS_STANDARD=-O2
-CFLAGS_DEBUG=-g -O
-CFLAGS_PROFILE=-pg -O2
-CFLAGS=$(CFLAGS_DEBUG) $(GCFLAGS) $(XCFLAGS)
+CFLAGS=$(GCFLAGS) $(XCFLAGS)
 LDFLAGS=$(XLDFLAGS)
 STDLIBS=-lpthread -lm
 EXTRALIBS=
